.jshintrc

```diff
@@ -13,7 +13,7 @@
     "newcap"        : false,    // true: Require capitalization of all constructor functions e.g. `new F()`
     "noarg"         : true,     // true: Prohibit use of `arguments.caller` and `arguments.callee`
     "noempty"       : true,     // true: Prohibit use of empty blocks
-    "nonew"         : true,     // true: Prohibit use of constructors for side-effects (without assignment)
+    "nonew"         : false,     // true: Prohibit use of constructors for side-effects (without assignment)
```

Comment:
https://jslinterrors.com/do-not-use-new-for-side-effects
lint was not allowing me to write new oc.Registry(args); without assigning to a variable.